bears/ruby/RuboCopBear.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class RuboCopBear(LocalBear, Lint):
+    executable = 'ruby'
+    arguments = '-S rubocop --format emacs {filename}'
+    output_regex = re.compile(
+        r'(?P<file_name>^.+?):\s*(?P<line>\d+):\s*'
+        r'(?P<col>\d+):\s*(:?(?P<severity>[RCW]|[EF])):\s*'
+        r'(?P<message>.+)')
+    severity_map = {
+        "W": RESULT_SEVERITY.NORMAL,
+        "C": RESULT_SEVERITY.NORMAL,
+        "R": RESULT_SEVERITY.NORMAL,
+        "E": RESULT_SEVERITY.MAJOR,
+        "F": RESULT_SEVERITY.MAJOR
+        }
+
```

Comment:
Align severity in decreasing order `MAJOR => NORMAL => INFO`, It makes things easier.